apps/testing-javascript/src/templates/lesson-template.tsx

```diff
@@ -41,6 +41,8 @@ const LessonTemplate = () => {
     <VideoProvider
       muxPlayerRef={muxPlayerRef}
       exerciseSlug={router.query.lesson as string}
+      handleContinue={customContinueHandler}
+      handlePlayFromBeginning={customPlayFromBeginningHandler}
```

Comment:
here we can provide overrides and sensible default is applied, this means we do it the same and call same handlers for all products